components/rich_text_editor/rich_text_editor.vue

```diff
@@ -283,3 +299,13 @@ export default {
   },
 };
 </script>
+
+<style lang="less">
+.ProseMirror p.is-editor-empty:first-child::before {
```

Comment:
Hmm, I see that this is what the extension creators are suggesting, so I guess it's fine... The indentation looks off though.